scripts/linters/general_purpose_linter.py

```diff
@@ -297,6 +297,12 @@
                    'for this purpose.',
         'excluded_files': (),
         'excluded_dirs': ()
+    },
+    {
+        'regexp': re.compile(r'\s+style\s*=\s*"[^\"]*"'),
```

Comment:
Can you please explain the last part of the regex (after =\s*)?

Reply:
I have changed it now, After the change I made it ignore regex with `<[]>` and `\"` inside style tag. I think these types of styles are part of angular.
![Screenshot from 2020-06-22 18-44-05](https://user-images.githubusercontent.com/29057268/85395837-61cbd600-b56e-11ea-9840-cb4caa3e53a3.png)